dspace-api/src/main/java/org/dspace/content/crosswalk/PREMISCrosswalk.java

```diff
@@ -264,7 +274,7 @@ public Element disseminateElement(Context context, DSpaceObject dso)
         {
             oiv.setText(baseUrl
                     + "/bitstream/"
-                    + URLEncoder.encode(handle, "UTF-8")
+                    + handle
```

Comment:
It's unclear to me why this URL encoding line was removed.  Could you explain this further?

Reply:
This does not _directly_ pertain to the PR's new functionality, but this was causing a problem PREMIS URL's. Encoding the handle results in a bad URL. e.g. `http://localhost:8080/bitstream/123456789%2F5/2/license.txt`

Perhaps the handle suffix/prefix should be UTF-8 encoded separately?